lms/djangoapps/certificates/apis/v0/views.py

```diff
@@ -21,7 +21,7 @@
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from openedx.core.djangoapps.user_api.accounts.api import visible_fields
 from openedx.core.lib.api.authentication import OAuth2AuthenticationAllowInactiveUser
-from openedx.core.openapi import swagger_auto_schema, openapi
+from openedx.core import openapi
```

Comment:
I liked the idea that developers do not have to know whether it's "openapi", "swagger", or any other particular system which is in charge of documenting the endpoints. They just add a decorator and expect that the docs will be automatically generated, somehow. This is why I was proposing "document_api_view", which is a more "neutral" name for the decorator (see PR #21816). Also, the tool-agnostic name ensures that we will not have to rename the decorator whenever we replace the tool.

Reply:
Yeah, I agree that "neutral" naming is good. The decorator names in this PR are neutral (`schema`, `parameter`), but the module name isn't. Even in your PR, we can't stop users from typing `from openedx.core import openapi; @openapi.document_api_view` if they choose to.

Perhaps the module name itself should be more neutral, like `apidoc`. That way, you'd end up with `apidoc.schema` or `apidoc.document_api_view`, and leave out references to OpenAPI.